static/app/views/issueDetails/groupRelatedIssues/index.tsx

```diff
@@ -111,14 +121,23 @@ function GroupRelatedIssues({params}: Props) {
                 <Title>{t('Trace connected issues')}</Title>
                 {traceConnectedIssues.length > 0 ? (
                   <div>
-                    <small>
-                      {t('These are the issues belonging to ')}
-                      <Link
-                        to={`/performance/trace/${traceMeta.trace_id}/?node=error-${traceMeta.event_id}`}
+                    <TextButtonWrapper>
+                      <small>
+                        {t('These are the issues belonging to ')}
+                        <Link
+                          to={`/performance/trace/${traceMeta.trace_id}/?node=error-${traceMeta.event_id}`}
```

Comment:
org slug

Reply:
Is there a way that component would require it? or that the docs would indicate it?

Link to source:
https://github.com/getsentry/sentry/blob/2fc9266e5d96ad2c82402ff0990877fc551753c2/static/app/components/links/link.tsx#L56-L61

Nothing in the hover helps me to know this:
<img width="941" alt="image" src="https://github.com/getsentry/sentry/assets/44410/ba272929-1ef7-47b6-b1c5-d0c930dcd7b5">

Reply:
@scttcper explained to me that it's important to prefix the org slug because of Hybrid Cloud